### ISSUE TITLE
About Download medium, small size.

### ISSUE THREAD

**Issue by SerenaButler** [LycheeOrg/Lychee-Laravel#98]
I think that is not the way to go. At least I don't see the sense :-P Users needs different sizes for e.g. Forum Posts, Shares etc, so there should be linkable URLs to select.

![image](https://user-images.githubusercontent.com/35893306/53585911-4d98e400-3b87-11e9-950b-60090c304526.png)

**Comment by kamil4** [comment]
I think "Full Photo" does what you want, only for the original file, correct? If so, adding such links for other variants (with HiDPI fully enabled, there could be up to six) should be trivial...

**Comment by d7415** [comment]
This would break if we block hotlinking (v3 has a plugin, I can't remember the progress on Laravel). It could still be added, but might have to be disabled if hotlinking prevention is enabled.

**Comment by SerenaButler** [comment]
I would propose the following (just a quick mockup). 

<img width="1502" alt="new wireframe 1 copy" src="https://user-images.githubusercontent.com/35893306/53630783-2686f480-3c11-11e9-8c4d-69398c5a2ceb.png">

**Comment by kamil4** [comment]
Exactly. I wonder if this can be put in the menu or if it needs to be a separate view (which would be a lot more work I guess)?
It could actually be up to seven image variants (thumb, thumb@2x, small, small@2x, medium, medium@2x, big). For each variant I would suggest listing its size (we didn't used to have that info in the frontend for small and medium images, but with the recent HiDPI changes, we now do), and providing for each a "copy URL to clipboard" button, a download button, and an "open in new window/tab" button.

**Comment by d7415** [comment]
I'd think a link that you can open or "save link as" would be enough rather than a download button. Laravel didn't support that anyway last I checked, and we're specifically talking about resized versions.

Honestly I mostly think this is overkill anyway, but there seem to be at least a few interested, so *shrug*
